regression-test/data/nereids_tpcds_shape_sf1000_p0/shape/query14.out

```diff
@@ -61,34 +61,23 @@ PhysicalCteAnchor ( cteId=CTEId#0 )
 ------hashAgg[GLOBAL]
```

Comment:
can we merge these two rules in one pr to avoid the shape change and better to track?